Tests/SwiftCliCoreTests/InputValidatorTest.swift

```diff
@@ -14,7 +14,16 @@ class ValidatorTests: XCTestCase {
 
     func testInvalidStitchShouldThrowError() throws {
         let testPattern = ["g1 p1"]
-        let err = InputError.invalidStitch(invalidStitch: "g1", rowLocation: 1)
+        let err = InputError.multipleErrors
+        (errors:
+            [
+                SwiftCliCore.InputError.invalidStitch(
+                    invalidStitch: "g1",
+                    rowLocation: Optional(1),
+                    stitchIndexInRow: Optional(1)
+                )
+            ]
+        )
```

Comment:
I had to stare at this quite a bit, wondering what the parentheses were doing. I think you mean 

```swift
let err = InputError.multipleErrors(
    errors: [
        ...
    ]
)
```